IBM/carbon-addons-iot-react
# fix(tablecard): added explicit check for 0

Closes #1015 

**Summary**

- The integer 0 was being treated as falsy, which caused the 0 to be omitted. This PR adds an explicit check for `=== 0`

**Change List (commits, features, bugs, etc)**

- Added 0 values to mock table data for future testing
- Added explicit check for integer 0 in CSV utility function
- Added test for use-case
- Added more comments about the utility functions
- Updated old test to reflect new 0 value
- Updated snapshots

**Acceptance Test (how to verify the PR)**

1. Go to `/?path=/story/watson-iot-tablecard--size-large` and download the CSV
2. See that the 0 values appear in the pressure column

Old:
<img width="588" alt="Screen Shot 2020-03-18 at 12 18 09 PM" src="https://user-images.githubusercontent.com/25177649/76995020-3e650f80-691d-11ea-8f4c-08a5269839e7.png">


Updated:
<img width="652" alt="Screen Shot 2020-03-18 at 1 33 37 PM" src="https://user-images.githubusercontent.com/25177649/76994991-2f7e5d00-691d-11ea-9701-c9c1f14632cb.png">
